database/seed.js

```diff
@@ -1,27 +1,32 @@
 let mongoose = require('mongoose');
 let Bookings = require('./model.js');
-var data = JSON.parse(require('./allListings.json'));
-
-var fakeAvailableDays = ['2/28/2018', '3/4/2018', '3/5/2018', 
-  '3/7/2018', '3/18/2018', '3/19/2018', 
-  '3/20/2018', '3/21/2018', '3/22/2018', 
-  '3/23/2018', '4/9/2018', '4/10/2018'
+let data = JSON.parse(require('./allListings.json'));
+let Promise = require('bluebird');
+
+var sampleAvailableDays = [
+  new Date(2018, 2, 7), new Date(2018, 2, 8), new Date(2018, 2, 9), 
+  new Date(2018, 2, 11), new Date(2018, 2, 12), new Date(2018, 2, 13),
+  new Date(2018, 2, 15), new Date(2018, 2, 16), new Date(2018, 2, 19),
+  new Date(2018, 2, 20), new Date(2018, 2, 21), new Date(2018, 2, 22),
+  new Date(2018, 2, 25), new Date(2018, 2, 26), new Date(2018, 3, 1),
+  new Date(2018, 3, 2), new Date(2018, 3, 3), new Date(2018, 3, 4),
+  new Date(2018, 3, 5), new Date(2018, 3, 6), new Date(2018, 3, 7),
+  new Date(2018, 3, 8), new Date(2018, 3, 9), new Date(2018, 3, 10),
+  new Date(2018, 3, 11), new Date(2018, 3, 12), new Date(2018, 3, 13),
 ];
 
-var insertOne = function(data) {
-  return BookingModel.create(data);
-};
 
 var seedDb = function(data) {
   mongoose.connect('mongodb://localhost/airbnb_bookings')
-    .then(connection => {
+    .then((connection) => {
+      connect = connection;
```

Comment:
I don't see this variable declared before being used. I'm surprised the linter didn't complain about it. You may wan to check out your editor/linter configuration